hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_exchange.py

```diff
@@ -249,9 +250,18 @@ async def _place_order(
         side = api.Side.S_BID if trade_type == TradeType.BUY else api.Side.S_ASK
         type = api.OrderType.OT_LIMIT if order_type == OrderType.LIMIT else api.OrderType.OT_MARKET
 
+        tokens = trading_pair.split("-")
+        base = tokens[0]
+        quote = tokens[1]
+
+        # this is temporarily hard coded to a single solana wallet
+
+        base_addr = CONSTANTS.TOKENPAIRTOWALLETADDR[base]
```

Comment:
I think the owner address should by default be the SOL address, but from what I remember, the payer address is dependent on the side?

Reply:
So if you are buying SOL/paying USDC, it should be payerAddress of USDC (like what you did)

But if you are paying/selling SOL, it should be payerAddress of SOL?